server/src/main/java/io/crate/profile/ProfilingContext.java

```diff
@@ -63,8 +67,15 @@ public Map<String, Object> getDurationInMSByTimer() {
         return Collections.unmodifiableMap(builder);
     }
 
-    private static Map<String, Object> resultAsMap(ProfileResult profileResult) {
+    private static Map<String, Object> resultAsMap(ShardId shardId, ProfileResult profileResult) {
         HashMap<String, Object> queryTimingsBuilder = new HashMap<>();
```

Comment:
thx @seut!

Out of quriosity, is the order kept somehow for those attributes (existing and new)?
Maybe we need a `LinkedHashMap`?

(Didn't dig any further)

Reply:
No the order isn't kept. Do you think the order is important here?

Reply:
Just for the shake of having a consistent output, since now there more fields added, it can be a bit more confusing to see those entries in mixed order (between different shards (and partitions).

Reply:
What about sorting it alphabetically once all is added? This seems to be done by the `crash` output anyhow and it looks fine to me.

Reply:
thx for checking!